Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM nvidia/cuda:12.6.0-runtime-ubuntu22.04
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PYTHONUNBUFFERED=1
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    python3.10 \
+    python3-pip \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Python packages
+RUN pip3 install --no-cache-dir \
```

Comment:
Lets add all the requirements to requirements.txt, so it can be installed and managed in one place in case user wants to use it. And this would be also useful in case users wants to test notebooks locally.

Reply:
This hasn't been addressed